gimie/project.py

```diff
@@ -43,8 +39,12 @@ class Project:
     base_url:
         The base URL of the git remote. Can be used to
         specify delimitation between base URL and project name.
-    sources:
-        The metadata sources to use.
+    git_provider:
+        The name of the git provider to extract metadata from.
+        ('git', 'github', 'gitlab')
+    parser_names:
+        Names of file parsers to use. ('license').
+        If None, all parsers are used.
```

Comment:
This works atm for the License Parser. Based on future parsers, we might want to some parsers optional and others by default

Reply:
Agreed. but as this is not yet an issue, perhaps we can keep it for a separate PR? :grin:

Reply:
Added `DEFAULT_PARSERS` and `PARSERS`.
All functions now use `DEFAULT_PARSERS` by default.